tests-utils/src/main/scala/tech/beshu/ror/utils/elasticsearch/BaseManager.scala

```diff
@@ -94,17 +96,26 @@ abstract class BaseManager(client: RestClient,
     override def toString: String = response.toString
 
     private def checkResponseAssertions(): Unit = {
-      if(!isForbidden && esNativeApi) {
-        if (Version.greaterOrEqualThan(esVersion, 7, 14, 0) && Version.lowerThan(esVersion, 7, 16, 0)) {
-          request match {
-            case Some(req) if isExcludedRequest(req) =>
-              // ES [7.14.0,7.16.0) doesn't add the X-elastic-product header to some responses (under some conditions)
-            case Some(_) | None =>
+      OsUtils.currentOs match {
+        case CurrentOs.Windows =>
+          // On Windows the header is not added to some requests in versions <7.14-8.0)
```

Comment:
This is bad. 
Let's create a jira task for this, and let's add a todo here with the jira ref.